Lib/functools.py

```diff
@@ -238,12 +238,14 @@ def reduce(function, sequence, initial=_initial_missing):
     """
     reduce(function, iterable[, initial], /) -> value
 
-    Apply a function of two arguments cumulatively to the items of a sequence
-    or iterable, from left to right, so as to reduce the iterable to a single
-    value.  For example, reduce(lambda x, y: x+y, [1, 2, 3, 4, 5]) calculates
-    ((((1+2)+3)+4)+5).  If initial is present, it is placed before the items
-    of the iterable in the calculation, and serves as a default when the
-    iterable is empty.
+    Apply a function of two arguments cumulatively to an iterable, from left to right.
+
+    This efficiently reduces the iterable to a single value.  If initial is present,
```

Comment:
Did you mean "effectively"? (No offence!)

```suggestion
    This effectively reduces the iterable to a single value.  If initial is present,
```